mon-pix/tests/acceptance/profilv2-test.js

```diff
@@ -32,10 +32,28 @@ describe('Acceptance | Profil v2 | Afficher profil v2', function() {
       await visit('/profilv2');
 
       // then
-      return andThen(() => {
-        expect(currentURL()).to.equal('/profilv2');
+      expect(currentURL()).to.equal('/profilv2');
+    });
+
+    describe('Display scorecards', function() {
+
+      it('should display last competence card', async function() {
+        // given
+        // when
+        await visit('/profilv2');
+        const lastContentArea = find('.scorecard:last-of-type .scorecard-content__area').text();
+
+        // then
+        expect(lastContentArea).to.contains('Création de contenu');
```

Comment:
to.equal ?

Reply:
Pour le moment, on a mis .contains parce que le find retourne `Area: <stuff>` pour des raisons de lisibilité

Reply:
en ce cas `expect(lastContentArea.title).to.equal('Création de contenu');` ?